guides/source/action_view_overview.md

```diff
@@ -1505,6 +1505,624 @@ strip_tags("<b>Bold</b> no more!  <a href='more.html'>See more</a>")
 
 NB: The output may still contain unescaped '<', '>', '&' characters and confuse browsers.
 
+### UrlHelper
```

Comment:
A lot of this is redundant to the [API docs we already maintain](https://api.rubyonrails.org/v6.0/classes/ActionView/Helpers/UrlHelper.html). Having this information in two places doesn't seem like a good solution, but I like the addition of mentioning URL helper here. I think it would make more sense if we provided a less detailed summary of the main helper methods provided by this module (like we are doing with other helpers). WDYT?

Reply:
The guide is already long and the intention isn't to document every helper. Let's cut this to just `link_to` and `button_to`, that's what most people will be using.

Reply:
@gmcgibbon I like the idea to cut details to summary. 

@kaspth other methods are almost unknown to most of the `Rails` users. so I think at least we should let them know that these APIs exists. May be we can reduce the detail in doc.

Reply:
> so I think at least we should let them know that these APIs exists.

I thought they were documented on api.rubyonrails.org? Maybe the guide can link to the corresponding documentation for the helper modules on api.rubyonrails.org. That might be a better route than keeping every bit of documentation in both places.

And since this guide is about giving an overview, I'm still not convinced we should document more than `link_to` and `button_to`. Since the guide is also to a certain extent about displaying idiomatic Rails code, we should just stick to the most common examples (e.g. no `controller: "searches"`, which while it works using routing helpers is the ~ 95%+ case and is generally preferred).

Reply:
@kaspth @gmcgibbon I reduced a lot. please review. I think we should keep this much.